modules/cnf-du-cnf-du-workload-partitioning-pod-mutation.adoc

```diff
@@ -0,0 +1,62 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-provisioning-and-deploying-a-distributed-unit.adoc
+
+[id="cnf-du-workload-partitioning-pod-mutation_{context}"]
+
+= Workload partitioning pod mutation
+
+The workload partitioning feature modifies pods that are annotated with `target.workload.openshift.io/<workload-type>`
+that are in a namespace that contains the `workload.openshift.io/allowed: <workload-type>` annotation.
+
+For example, a pod with the following CPU request specified would be mutated:
+
+[source,yaml]
+----
+requests:
+  cpu:
+    400m
+----
+
+The requested CPU resource is replaced with a management cores resource:
+
+[source,yaml]
+----
+requests:
+  management.workload.openshift.io/cores: 400
+----
+
+The value of `resources.workload.openshift.io/{container-name}` is calculated as:
+
+----
+shares == (request_in_millis * 1024) /1000
+----
+
+This results in a value of 409 in this example.
+
+An annotation is added with the same value:
+
+[source,yaml]
+----
+annotations:
+  resources.workload.openshift.io/{container-name}: {"cpushares": 409}
+----
+
+The new request value and annotation value are scaled up by 1000 from the original CPU request because
+opaque resources do not support units or fractional values. In the preceding example, the request is in
+milli-cores (400m) and the modified value is unitless.
+
+Pods are not mutated in a way that changes their Quality of Service (QoS) class. For example, the feature does not mutate CPU requests unless the pod also has memory requests, because if you mutate the pod so that it has no CPU or memory requests the quality-of-service class of the pod would be changed automatically. In this case, the `workload.openshift.io/warning` annotation includes a message explaining that the partitioning instructions were ignored. Any pod that is already BestEffort is annotated using `2` as the value so that CRI-O has an indicator to configure the CPU shares as `BestEffort`.
```

Comment:
Please replace all "quality-of-service" with "Quality of Service".  This will match the first instance.